CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+docs/CONTRIBUTING.md
```

Comment:
So, I moved `CONTRIBUTING.md` to the docs site. I think it's a good idea for discovery to have it in the root directory too (if only for people to quickly find them if browsing the repo on GitHub), so I tried to symlink it. VSCode is able to show me the content of the symlinked file, but GitHub just shows this file as is. Do you have an idea on how to make GitHub follow the symlink?

Reply:
I checked out the `docs/contributing` branch, and the symlink looked fine (I confirmed it was relative). I don't think there is a way for GitHub to render the contents rather than the symlink info. We could add a makefile target (`make docs`) that runs `cp docs/CONTRIBUTING.md CONTRIBUTING.md`. Another option would be to add a link at the bottom of the main `README.md` to the contributing section of https://cs01.github.io/termpair/, which seems a little more straightforward.

Reply:
I do think a link to documentation should be added to the top of the main README. If that is there, I think it's reasonable that someone who wants to contribute would click on that and find the contributing section.

Reply:
Yup. The solution of adding a link to the docs seems fine to me. Will add this tomorrow hopefully!

Reply:
I pushed an update that adds a link to the docs in the repo's `README`, as well as some fixes to how images were centered, and clearer sections for the `README` and the docs home page. Let me know what you think!